src-electron/generator/helper-zcl.js

```diff
@@ -528,18 +529,24 @@ async function zcl_commands_with_cluster_info(options) {
 async function zcl_commands_with_arguments(options) {
   let sortBy = options.hash.sortBy
   let packageIds = await templateUtil.ensureZclPackageIds(this)
+  let packages = await queryPackage.getPackagesByPackageIds(
+    this.global.db,
+    packageIds
+  )
 
   let cmds = await Promise.all(
-    packageIds.map(async (packageId) => {
+    packages.map(async (pkg) => {
       let cmdsPerPackageId = await queryCommand.selectAllCommandsWithArguments(
```

Comment:
what is the point of this "await"?

Reply:
The one before "Promise.all()"? You need to make sure each individual promise resolves.